vocabularies/Hierarchy.xml

```diff
@@ -145,6 +145,44 @@ the following collections of hierarchy nodes are distinguished:
         -->
       </ComplexType>
 
+      <Term Name="RecursiveHierarchyActions" Type="Hierarchy.RecursiveHierarchyActionsType" BaseTerm="Aggregation.RecursiveHierarchy" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Actions for maintaining the recursive hierarchy defined by the [base term](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.html#RecursiveHierarchy)" />
+        <Annotation Term="Core.LongDescription">
+          <String>When this annotation is present, the [`ParentNavigationProperty`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.html#RecursiveHierarchyType)
+          in the base term must not have a collection-valued segment prior to its last segment.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="RecursiveHierarchyActionsType">
+        <Annotation Term="Core.LongDescription">
+          <String>The qualified action names identify actions for maintaining nodes in the recursive hierarchy,
+which are specific for the given annotation qualifier.
+These actions MUST have the same signature as the template actions linked below, with
+`Edm.EntityType` replaced with the entity type on which the recursive hierarchy is defined.
+In parameters that are defined as "Key of …", only the key properties are relevant.
+
+The template actions themselves cannot be invoked.</String>
+        </Annotation>
+        <Property Name="MakeStartNodeAction" Type="Core.QualifiedActionName" Nullable="true">
```

Comment:
```suggestion
        <Property Name="MakeStartNodeAction" Type="Edm.String" Nullable="true">
```
Since we disallow aliases here, we cannot use the type `Core.QualifiedActionName`.